src/sentry/api/endpoints/organization_repositories.py

```diff
@@ -24,6 +32,11 @@ def get(self, request, organization):
         :pparam string organization_slug: the organization short name
         :auth: required
         """
+        if not self.has_feature(request, organization):
+            return Response({
+                'error_type': 'unavailable_feature',
+            }, status=400)
```

Comment:
I don't think 400 is a correct response here, since this is more like permission denied, so I'd suggest 403. But I don't think we really have a precedent. 400 is generally just used for like, malformed requests. Say, invalid JSON request body or something. Or invalid HTTP entirely.

Reply:
I'll do 403, but 400 is also used for any kind of invalid request (e.g. this param is not valid), not just less common things like parsing errors.